go.mod

```diff
@@ -54,4 +54,5 @@ replace (
 	k8s.io/apiserver => k8s.io/apiserver v0.17.6
 	k8s.io/client-go => k8s.io/client-go v0.17.6
 	k8s.io/code-generator => k8s.io/code-generator v0.17.6
+	knative.dev/eventing => github.com/lionelvillard/eventing v0.5.1-0.20200601200520-c1696d4abfb4
```

Comment:
Let's get rid of this (and others in modules.txt and go.sum) and we're good to go